doc_src/set_color.txt

```diff
@@ -2,36 +2,35 @@
 
 \subsection set_color-synopsis Synopsis
 \fish{synopsis}
-set_color [OPTIONS] [COLOR]
+`set_color` [OPTIONS] *VALUE*
```

Comment:
These backticks and asterisks are showing up in the man page and the docs. Were you trying to get some particular styling here?